cmake/findDependencies.cmake

```diff
@@ -46,6 +46,10 @@ if( EXIV2_ENABLE_PNG )
     find_package( ZLIB REQUIRED )
 endif( )
 
+if( EXIV2_ENABLE_BMFF )
+    find_package( Brotli )
```

Comment:
[question] If I understand this correctly, you are looking for the Brotli library optionally when BMFF is turned on, right?

Reply:
Yes.

Reply:
Now changed to be controllable by user.